src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -0,0 +1,150 @@
+                               theme)}}
+   (if (some more-icon-for-sizes [size])
+     [quo2.icons/icon :i/more
+      {:size  12
+       :color (get-overflow-color
+               blur?
+               colors/neutral-80-opa-70
+               colors/white-opa-70
+               colors/neutral-50
+               colors/neutral-40
+               theme)}]
+     [quo2.text/text
+      {:size   (if (= size 32) :paragraph-2 :label)
+       :weight :medium
+       :style  {:color       (get-overflow-color
+                              blur?
+                              colors/neutral-80-opa-70
+                              colors/white-opa-70
+                              colors/neutral-50
+                              colors/neutral-40
+                              theme)
+                :margin-left -2}}
+      ;; If overflow label is below 100, show label as +label (ex. +30), else just show 99+
+      (if (< label 100)
+        (str "+" label)
+        more-than-99-label)])])
+
+(defn- preview-item
+  [{:keys [item type size-key]}]
+  (let [size             (get-in properties/sizes [size-key :size])
+        user-avatar-size (get-in properties/sizes [size-key :user-avatar-size])
+        border-radius    (get-in properties/sizes
+                                 [size-key :border-radius (properties/border-type type)])]
+    (case type
+      :user                        [user-avatar/user-avatar
+                                    (merge {:ring?             false
+                                            :status-indicator? false
+                                            :size              user-avatar-size}
+                                           item)]
+
+      :accounts                    [account-avatar/view
+                                    (merge item {:size size})]
+
+      (:communities :collectibles) [fast-image/fast-image
+                                    {:source (:source item)
+                                     :style  {:width         size
+                                              :height        size
+                                              :border-radius border-radius}}]
+
+      (:tokens :network :dapps)    [fast-image/fast-image
+                                    {:source (:source item)
+                                     :style  {:width         size
+                                              :height        size
+                                              :border-radius border-radius}}]
+      nil)))
+
+(defn- list-item
+  [{:keys [index type size-key item number]}]
+  (let [last-item?  (= index (- number 1))
+        margin-left (get-in properties/sizes [size-key :margin-left])
+        hole-size   (get-in properties/sizes [size-key :hole-size])
+        hole-radius (get-in properties/sizes [size-key :hole-radius (properties/border-type type)])
+        hole-x      (get-in properties/sizes [size-key :hole-x])
+        hole-y      (get-in properties/sizes [size-key :hole-y])]
```

Comment:
Instead of performing multiple nested key extractions (`get-in` calls), we could use a destructuring and a single extraction :thinking: 
I'm not sure about code readability, but wdyt about this:

change:
```clojure
(let [margin-left (get-in properties/sizes [size-key :margin-left])
      hole-size   (get-in properties/sizes [size-key :hole-size])
      hole-radius (get-in properties/sizes [size-key :hole-radius (properties/border-type type)])
      hole-x      (get-in properties/sizes [size-key :hole-x])
      hole-y      (get-in properties/sizes [size-key :hole-y])]
     ,,,)
```
for:
```clojure
(let [border-type                              (properties/border-type type)
      {margin-left               :margin-left
       hole-size                 :hole-size
       hole-x                    :hole-x
       hole-y                    :hole-y
       {hole-radius border-type} :hole-radius} (properties/sizes size)]
  ,,,
  )
```

Reply:
Looks good 👍